### ISSUE TITLE
Design guide

### ISSUE THREAD

**Issue by EbenSorkin** [NeilSureshPatel/font-lang-support-afr#15]
I mention this idea not because it is a natural part of this project but because your helpful advice about Bhook Vhook Esh and Ezh reminded me that I think font designers like me could benefit from even more.

It would be great to have a design guide discussing how to design the less familiar glyphs discussing: contrast, gesture, proportion, terminals, anchor placement, and relation of the novel forms to parts of more familiar letter forms. 

And it would be nice if this advice was transparent about the sources and rationales that inform these opinions.

I think the glyphs ( letters and diacritics ) that would need addressing if this was done for the global glyph set then it might include: 
UC - Alpha-latin Astroke Aturned Bhook Bstroke Btopbar Chook Cstroke Dtopbar Eopen Ereversed EreversedOpen Esh Estroke Schwa Glottalstop Gstroke Yogh Heng Hhook Hturned Iota-latin Istroke Khook Kstroke Lbar Lbelt Ldoublebar Lmiddletilde Mhook Mturned Nhookleft Nlongrightleg Eng OU Omega-latin Oopen Phook Pstroke Qhooktail Rstroke Rtail Sobliquestroke Tbar Tdiagonalstroke Thook Tretroflexhook Ubar Upsilon-latin Gamma-latin Vhook Vturned Whook Yhook Aogonek.loclNAV Eogonek.loclNAV Iogonek.loclNAV Oogonek.loclNAV Uogonek.loclNAV Eng.loclNSM Vhook.loclTOD0 Bhook.loclXPE  

Lc - Ismall Nsmall Rsmall Usmallstroke alpha-latin arighthalfring astroke beta-latin bhook bilabialclick btopbar cbar chi-latin chook creversed dhook dhookandtail dtail dtopbar eopen eopenreversed esh estroke ezhcaron ezhreversed gamma-latin ghook glottalstop glottalstopreversed glottalstopsmall yogh heng hhook hturned iota-latin jcrossedtail jstroke khook kstroke lambdastroke lbar lbelt ldoublebar lmiddletilde longsdotaccent mhook mturned nhookleft nlegrightlong obarred omega-latin oopen ou phook pstroke qhooktail ramshorn rfishhook rstroke rtail sobliquestroke longs tdiagonalstroke thook tretroflexhook tturned ubar upsilon-latin vhook vturned whook yhook ystroke yturned zstroke saltillo istroke.dotless aogonek.loclNAV eogonek.loclNAV iogonek.loclNAV oogonek.loclNAV uogonek.loclNAV gstroke 

And the relvant sc 

 I have started to collect my own version of this just as a spreadsheet. But I am very aware of how limited my view is.

**Comment by NeilSureshPatel** [comment]
I agree, this would be a really great addition, but I don't think we have enough info to accomplish this yet. Most of the orthographic data is collected from published sources like National orthographies or dictionaries, etc. rather than manuscripts. This makes it difficult to have a clear view of specific preferences for the forms. It is possible that SIL had gathered manuscript resources when they compiled their orthographic lists but those original sources are not as easy to come by nor does SIL provide anything more specific on ScriptSource than we do in this project. To determine specific design preferences it is probably best to look at a number of manuscripts as not to focus too specifically on an individual's personal style. To really get a good picture of this, I think, will require a fair amount of field work since a lot of published content and manuscripts are not in libraries. Hopefully over time more of this data will be available and then can be included in this project. 

I will say that I think in general it is best to design the letters in question using the design language of your fonts so they fit in. The point of of the GF library is to offer a breadth of styles to users. If something doesn't click with users hopefully another font does. I know this sounds a bit backwards but I think getting language support online and in Google Docs will go a long way to making making more language specific resources publicly available. Basically making things work well enough so that we can learn more and improve fonts and support.
